File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/project/GraduationDesignPlanBean.java
package top.zbeboy.isy.web.bean.graduate.design.project;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.GraduationDesignPlan;

/**
 * Created by zbeboy on 2017/6/1.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class GraduationDesignPlanBean extends GraduationDesignPlan {
    private String buildingName;
    private String buildingCode;
    private int buildingId;
}

File: src/main/java/top/zbeboy/isy/web/bean/graduate/design/project/GraduationDesignPlanBean.kt
package top.zbeboy.isy.web.bean.graduate.design.project

import top.zbeboy.isy.domain.tables.pojos.GraduationDesignPlan

/**
 * Created by zbeboy 2017-12-20 .
 **/
class GraduationDesignPlanBean: GraduationDesignPlan() {
    var buildingName: String? = null
    var buildingCode: String? = null
    var buildingId: Int = 0
}